fastify.js

```diff
@@ -271,15 +271,12 @@ function build (options) {
   }
 
   function _handleTrustProxy (req) {
-    req.ip = proxyAddr(req, proxyFn)
-    req.ips = proxyAddr.all(req, proxyFn)
-    if (req.ip !== undefined) {
-      req.hostname = req.headers['x-forwarded-host']
-    }
+    var ip = proxyAddr(req, proxyFn)
+    return { ip: ip, ips: proxyAddr.all(req, proxyFn), hostname: ip !== undefined ? req.headers['x-forwarded-host'] : undefined }
```

Comment:
Small nit-pick; for code readability can you new line each `key: value` pair?

Reply:
I don't understand this change.

Reply:
From:
```js
return { ip: ip, ips: proxyAddr.all(req, proxyFn), hostname: ip !== undefined ? req.headers['x-forwarded-host'] : undefined }
```
to:
```js
return { 
  ip: ip, 
  ips: proxyAddr.all(req, proxyFn), 
  hostname: ip !== undefined ? req.headers['x-forwarded-host'] : undefined 
}
```
With both a method call and a ternary expression, this code seems very cluttered all in one line. Totally just a preference and can be ignored

Reply:
Refactored the function that's determining `ip`, `ips`, and `hostname` in the trust-proxy scenario. [43657f8](https://github.com/fastify/fastify/commit/43657f8357db5ebe896d9909dd1d1b7ad5d3fd0e)